api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -305,10 +305,34 @@ message ClusterManager {
 // Envoy process watchdog configuration. When configured, this monitors for
 // nonresponsive threads and kills the process after the configured thresholds.
 // See the :ref:`watchdog documentation <operations_performance_watchdog>` for more information.
-// [#next-free-field: 7]
+// [#next-free-field: 8]
 message Watchdog {
   option (udpa.annotations.versioning).previous_message_type = "envoy.config.bootstrap.v2.Watchdog";
 
+  message WatchdogAction {
+    // The events are fired in this order: KILL, MULTIKILL, MEGAMISS, MISS.
+    // Within an event type, actions execute in the order they are configured.
+    // For KILL/MULTIKILL there is a default PANIC that will kill the process,
+    // but it might be useful to specify several debug actions, and possibly
```

Comment:
Sorry can you clarify this a bit? Does specifying any config for KILL/MULTIKILL override the default PANIC action? It's not clear from the doc.